### PR TITLE
Tarto forms

### PR DESCRIPTION
**Pull request by Tarto-4**
# Contact validation form:
## Applied Features:
- Created a validation form that accepts Email address entered at lowercase
- Included Error message if correct format is not followed by user
- Added styling to the Error message

### CODE REVIEW

**Comment by addod19** [comment]
### Hi @Tarto-4, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because the pair programming activity is done solo. We expect commit messages from both coding partners

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: form.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// CODE
+form.addEventListener('submit', (event) => {
+  // preventing from submittion
+  event.preventDefault();
+  const userInput = userEmail.value;
+  if (userInput !== userInput.toLowerCase()) {
+    error.innerText = errorMessage;
+  } else {
+    error.innerText = '';
+    form.submit();
```
Comment (GzimAsani) [inline form.js]: - [ ] You are having an error here whenever the form is good from validations but it cannot actually submit, kindly check it 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, TEAM           :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with adding the validation, it works great :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N/A



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ademibowale — APPROVED** [review]
Hi @Tarto-4  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- If a user submits the contact form with the email field containing uppercase characters an error message should show up.✔️ 
- If a user submits the contact form with the email field containing only lowercase characters the form is sent normally. ✔️ 
- The error message  show up close to the `submit` button or the `input fields`. (i.e error message show up around the input fields or just around the submit button)✔️
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**